front/src/modules/settings/data-model/components/SettingsObjectFieldRelationForm.tsx

```diff
@@ -18,6 +18,7 @@ export type SettingsObjectFieldRelationFormValues = Partial<{
 }>;
 
 type SettingsObjectFieldRelationFormProps = {
+  disableFieldEdition?: boolean;
```

Comment:
disableFieldNamesEdition?

Reply:
This disables the icon picker (for the field icon) and the input (for the field name), so it does not disable only the field name edition!